### PR TITLE
Documentation add timezone and localtime volumes to Docker Guide 

### PR DESCRIPTION
**Pull request by h00000000**
This pull request adds timezone and localtime volumes to the [Installation with Docker](https://docs.gitea.io/en-us/install-with-docker/) guide.
Passing these volumes to the container enable gitea to use hosts localtime and timezone.

This is a mandatory change when using the [Fail2ban setup to block users after failed login attemts](https://docs.gitea.io/en-us/fail2ban-setup/) Guide, but it is not described in the documentation. If not set up correctly, a user will experience the following fail2ban behaviour:

Testing with fail2ban-regex will report a valid regex filter.
`sudo fail2ban-regex ./gitea/log/gitea.log /etc/fail2ban/filter.d/gitea.conf`

On failed authentication attempt, in certain time configurations, fail2ban will not report the failed attempt, neither will increment the failure detection count or proceed to ban the offender after n tries.

This issue was reported in the past ([6421](https://github.com/go-gitea/gitea/issues/6421)), it seems the issue was forgotten but the described problems still apply today.

### CODE REVIEW

**Comment by huebnerf** [comment]
In question of the documentation change being added to the [docker guide](https://docs.gitea.io/en-us/install-with-docker/) or the [fail2ban guide](https://docs.gitea.io/en-us/fail2ban-setup/), like the original reported of the issue, i think that using hosts /etc/timezone and /etc/localtime in the container should be considered, even if not using fail2ban

**Review by techknowlogick — APPROVED** [review]

File: docs/content/doc/installation/with-docker.en-us.md [hunk 1]
```diff
@@ -209,6 +217,8 @@ services:
     volumes:
 -      - ./gitea:/data
 +      - gitea:/data
```
Comment (kolaente) [inline docs/content/doc/installation/with-docker.en-us.md:219]: Why did you change this?

Comment (techknowlogick) [reply docs/content/doc/installation/with-docker.en-us.md:219]: It’s not changed. The docs contain a code block for diff type and so GitHub renders the diff. If you “view file” rendered you’ll see the docs are recommending a change for users to make to their compose file.

**Review by mrsdizzie — APPROVED** [review]
